routers/books.js

```diff
@@ -103,14 +106,29 @@ bookRouter.put('/', (req, res) => {
  * @apiGroup Books
  * @apiVersion 0.1.0
  *
- *@apiParam {String} id Id of the Book being fetched.
+ *@apiParam {String} id Id of the Book being deleted.
  *
  * @apiSuccessExample {json} Success-Response:
- * 		HTTP/1.1 204 OK
+ * 		HTTP/1.1 204 No Content
  */
-bookRouter.delete('/', (req, res) => {
-	console.log('DELETE');
-	res.send('Got a DELETE request at /user');
+bookRouter.delete('/:id', [param('id').exists()], async (req, res) => {
+	try {
+		const errors = validationResult(req);
+		if (!errors.isEmpty()) {
+			return res.status(400).json({ errors: errors.array() });
+		}
+
+		const book = await bookDoc.findByIdAndRemove(req.params.id);
+
+		if (!book) {
+			return res.sendStatus(404);
+		}
+
+		res.sendStatus(204);
```

Comment:
Why do you return 204? Why not 200?

Reply:
isn't 204  HTTP the standard for a delete request?

Reply:
mhh do you have a resource that defines this? when we use 200, we can use it all the time and do not need to check for 204 or 200, makes life easier, what do you think?